src/applications/claims-status/tests/components/claim-files-tab/FilesNeeded.unit.spec.jsx

```diff
@@ -10,10 +11,37 @@ const item = {
 };
 
 describe('<FilesNeeded>', () => {
-  it('should render va-alert with item data', () => {
-    const screen = renderWithRouter(<FilesNeeded item={item} />);
-    screen.getByText(item.displayName);
-    screen.getByText(item.description);
-    screen.getByText('Details');
+  it('should render va-alert with item data and show DueDate', () => {
+    const { queryByText, getByText } = renderWithRouter(
+      <FilesNeeded item={item} />,
+    );
+    queryByText('December 1, 2024');
```

Comment:
So get by text wasnt letting me do this because it wanted the whole string of text EX: `Needed from you by April 7, 2024 - Due a month ago` I dont want to check that whole text I just want to check the date so thats why I did `queryByText`.

Reply:
Yeah so `queryByText` expects the whole string as well, the only reason your test isn't failing is because queryByText doesn't throw an error if the supplied text isn't found (which is why we use it instead of `getByText` when we want to assert that something isn't in the DOM). If you do `expect(queryByText('December 1, 2024')).to.exist;` your test will fail

Reply:
I think `getByText('December 1, 2024', { exact: false });` should work

Reply:
Thanks @jerekshoe that worked!

Reply:
Yeah that would work as well